cocos/platform/openharmony/AppDelegate.cpp

```diff
@@ -65,8 +65,10 @@ bool AppDelegate::applicationDidFinishLaunching()
     se->start();
     
     se::AutoHandleScope hs;
-    // jsb_run_script("jsb-adapter/jsb-builtin.js");
-    // jsb_run_script("main.js");
+    #if SCRIPT_ENGINE_TYPE == SCRIPT_ENGINE_V8
```

Comment:
```suggestion
    #if SCRIPT_ENGINE_TYPE != SCRIPT_ENGINE_NAPI
```
It's better to use `not napi`, since if we support other js engine backends, this check will break it.

Reply:
done